analytics/index.yaml

```diff
@@ -0,0 +1,727 @@
+            <p>Used in <em>Referrers</em> > <em>Campaigns</em> report.
+          type: string
+          #TODO: Is it possible to set multiple keywords to the campaign?
+          example: promotion
+        - name: res
+          in: formData
+          #TODO: Confirm used unit (pixels).
+          description: Resolution of Visitors device in pixels.
+          type: string
+          format: '{height}x{width}'
+          example: 1920x1080
+        - name: h
+          in: formData
+          description: Hour when the request was made in Visitors local time.
+          type: integer
+          minimum: 0
+          maximum: 23
+          example: 16
+        - name: m
+          in: formData
+          description: Minute when the request was made in Visitors local time.
+          type: integer
+          minimum: 0
+          maximum: 59
+          example: 44
+        - name: s
+          in: formData
+          description: Second when the request was made in Visitors local time.
+          type: integer
+          minimum: 0
+          maximum: 59
+          example: 12
+        - name: ua
+          in: formData
+          description: |
+            <p><em>User-Agent</em> value.</p>
+            <p>This value can be used to override value send in request HTTP header.</p>
+            <p>It's used to detect Visitors browser and operating system.</p>
+          type: string
+          example: |
+            Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.162 Safari/537.36
+        - name: lang
+          in: formData
+          #TODO: Do we support any instances without GeoIP? Is there any other reason to gather this data?
+          description: |
+            <p><em>Accept-Language</em> value.</p>
+            <p>This value can be used to override value send in request HTTP header.</p>
+            <p>It's used to guess Visitors country when GeoIP is disabled.</p>
+          type: string
+          example: en-US,en;q=0.9,pl;q=0.8
+        - name: uid
+          in: formData
+          #TODO: Check if Piwik passes this information to AM.
+          #TODO: Looks like this functionality is trying to implement basic user profile. Should it be depricated in favor of AM?
+          #TODO: Previous documentation is unclear and dispersed. We should link to a page describing how uid is used in detail.
+          description: |
+            <p>User ID.</p>
+            <p>Can be used to identify Visitor by external application (e.g. login name, email address or internal
+            user ID).</p>
+            <p>It allows merging visits accross different devices (e.g. laptop, tablet and smartphone).</p>
+          type: string
+          minLength: 1
+          example: PhilipFry
+        - name: cid
+          in: formData
+          #TODO: What's the difference between _uid, cid and uid parameters? Are some of those deprecated?
```

Comment:
This one overwrites `config_id` the latter one overwrites `idvisitor`

Reply:
OK, in that case we need to mention that in descriptions. It would be also useful to explain intention of each of those parameters. I see you changed description to "Config ID", but i still don't know what it means. You can link do explanation if it's described in help.piwik.pro.